src/core.js

```diff
@@ -876,6 +876,10 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     return numericData;
   }
 
+  function isNumericData(value) {
+    return value.length > 0 && /^-?[\d\s]*(\.|,)?\d*$/.test(value);
```

Comment:
is this regex already being used or are we introducing it? feels like it should conform to what HOT already considers as a number?